wordpress-mobile/WordPress-iOS
# Revisions: Revision display/browsing

Refs #10304 

This implements the basic revision viewing & browsing.

![revisions display mk2](https://user-images.githubusercontent.com/517257/48682528-7d365a00-eb5d-11e8-8d2f-5d4f55297520.gif)

To test:
 - Edit a post with revision history
 - Tap `…` then `History`
 - Tap a revision
 - Ensure the UI behaves as expected
